opal/mca/threads/pthreads/threads_pthreads_mutex.h

```diff
@@ -120,10 +127,9 @@ static inline int opal_mutex_trylock(opal_mutex_t *m)
 {
 #if OPAL_ENABLE_DEBUG
     int ret = pthread_mutex_trylock(&m->m_lock_pthread);
-    if (ret == EDEADLK) {
+    if (EDEADLK == ret) {
         errno = ret;
-        perror("opal_mutex_trylock()");
-        abort();
+        opal_output(0,"opal_mutex_trylock() %d",ret);
```

Comment:
You can't return EDEADLK here. You need to somehow convert to an OPAL error code. This comment is true everywhere in this file.

Reply:
Thanks for the comment. We missed it. The latest commit translated all error codes to OPAL ones (`OPAL_ERROR` and `OPAL_ERR_IN_ERRNO`).